test/compileSync.js

```diff
@@ -0,0 +1,54 @@
+var chai = require("chai")
+var spies = require("chai-spies");
+var path = require("path");
+var compiler = require(path.join(__dirname, ".."));
+
+chai.use(spies);
```

Comment:
Ahhhh ! You are right !
I thought I needed that to use the assertion on the `emitWarning` spy.
Removing !